master/internal/resourcemanagers/resource_pool_test.go

```diff
@@ -14,283 +12,108 @@ import (
 	cproto "github.com/determined-ai/determined/master/pkg/container"
 )
 
-func (rp *ResourcePool) addAllocatedTask(
-	req *AllocateRequest, allocated *ResourcesAllocated,
-) {
-	rp.taskList.AddTask(req)
-	rp.taskList.SetAllocations(req.TaskActor, allocated)
-}
-
-func (rp *ResourcePool) addAgent(
```

Comment:
Why did this code get moved and renamed?

Reply:
I think adding a new member function with an unclear name is bad. This removed function allows you to specify numbers of used slots and zero-slot containers without actually associating with mock task containers actors and task actors. It will make the state of the agents and tasks inconsistent. It would make the tests create some cases that would never happen in reality and be hard to understand. So I moved this function as a standalone function and rename it to be more precise.